frontend/src/components/Navbar.js

```diff
@@ -6,8 +6,7 @@ import {
   NavbarBrand,
   Nav,
   NavItem,
-  NavLink,
-  NavbarText
```

Comment:
Why did you remove NavbarText and not add it back?

Reply:
I removed importing the NavBarText because it was not being used. I got notifications from the console that it was being imported but not called.